geometry_msgs/Pose goal_pose
string act
---
bool response
